librdmacm/CMakeLists.txt

```diff
@@ -41,3 +41,7 @@ install(TARGETS rspreload DESTINATION "${CMAKE_INSTALL_LIBDIR}/rsocket/")
 # FIXME: Maybe we can get rid of them?
 rdma_install_symlink("librspreload.so" "${CMAKE_INSTALL_LIBDIR}/rsocket/librspreload.so.1")
 rdma_install_symlink("librspreload.so" "${CMAKE_INSTALL_LIBDIR}/rsocket/librspreload.so.1.0.0")
+
+set(PC_LIB_NAME "rdmacm")
+configure_file(${CMAKE_SOURCE_DIR}/buildlib/template.pc.in ${CMAKE_CURRENT_BINARY_DIR}/lib${PC_LIB_NAME}.pc @ONLY)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/lib${PC_LIB_NAME}.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
```

Comment:
1. Three lines below worth to put into separate function in buildlib/rdma_functions.cmake
2. I don't see "set(PC_REQUIRES "libibverbs")" line here, is it ok?

Thanks

Reply:
1) sure will do that
2) does it require it? I don't see rdmacm linking to it here?